code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -82,8 +83,17 @@
 /obj/structure/stool/bed/chair/wood
 	burn_state = FLAMMABLE
 	burntime = 20
+	buildstackamount = 3
+	buildstacktype = /obj/item/stack/sheet/wood
 	// TODO:  Special ash subtype that looks like charred chair legs
 
+/obj/structure/stool/bed/chair/wood/attack_animal(mob/living/simple_animal/user)

```

Comment:
no reason to override this; this is the same thing as parent.

FOrgot to remove it?